__tests__/endpoints.test.js

```diff
@@ -0,0 +1,16 @@
+const app = require('../app');
+const request = require('supertest');
+const endpointsData = require('../endpoints.json');
+
+describe('/api', () => {
+    test('should return an object containing all available endpoints', (done) => {
+        request(app)
+            .get('/api')
+            .expect(200)
+            .end((err, response) => {
+                if (err) return done(err);
+                expect(response.body).toEqual(endpointsData);
```

Comment:
Currently ```res.body``` is just an object with no context. It is a lot nicer to send the endpoints in an object with a key explaining what it is. In this case ```endpoints```